index.js.flow

```diff
@@ -3,4 +3,4 @@ interface ComponentSize {
   height: number,
 }
 
-declare export default function useComponentSize<T>(ref: React.Ref<T>): ComponentSize;
\ No newline at end of file
+declare export default function useComponentSize<T>(ref: T | React.Ref<T>): ComponentSize;
```

Comment:
Remove the `React.Ref<T>` type if we make this a breaking change

Reply:
separately, idk where the flow types are since idk how flow works, but is this supposed to be `RefObject`? *shrug*